### PR TITLE
Add Neural Electronic Interface Ontology

### PR DESCRIPTION
**Pull request by wdduncan**
Add redirect for the Neural Electronic Interface Ontology.

### CODE REVIEW

File: neural-electronic-interface-ontology/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+The Neural Electronic Interface Ontology represents the types of devices used to interface with the central nervous system to enhance (or augment) stimuli from a subject's external environment, the conditions under which these devices should be employed, the assessment of a subject's sensory ability, and the metrics for evaluating the performance.
+
+## Ontology IRI
+https://w3id.org/neural-electronic-interface-ontology/neio.owl
+
+## Namespace
+https://w3id.org/neural-electronic-interface-ontology/NEIO_
+
+## Contacts
+Bill Duncan: wdduncan@gmail.com
```
Comment (TallTed) [inline neural-electronic-interface-ontology/README.md:11]:
```suggestion
* Bill Duncan <wdduncan@gmail.com> ([@wdduncan](https://github.com/wdduncan))
```

**Comment by dgarijo** [comment]
@wdduncan please commit the suggestion by @TallTed, as we prefer having the github ids to check in case of future changes

**Comment by wdduncan** [comment]
> @wdduncan please commit the suggestion by @TallTed, as we prefer having the github ids to check in case of future changes

Done. Thanks!

**Comment by wdduncan** [comment]
@dgarijo Do you know how long before the redirect works? I'd like to know when I can test what I've done.

**Comment by dgarijo** [comment]
Usually like 30 secs or so

**Comment by wdduncan** [comment]
Hmmm ... my rewrites don't seem be working.  
`https://w3id.org/neural-electronic-interface-ontology/neio.owl` should redirect to [https://raw.githubusercontent.com/uflcod/neural-electronic-interface-ontology/main/neio.owl](https://raw.githubusercontent.com/uflcod/neural-electronic-interface-ontology/main/neio.owl)

And `https://w3id.org/neural-electronic-interface-ontology/NEIO_0000101` should redirect to [https://bioportal.bioontology.org/ontologies/NEIO/?p=classes&conceptid=https%3A%2F%2Fw3id.org%2Fneural-electronic-interface-ontology%2FNEIO_0000101](https://bioportal.bioontology.org/ontologies/NEIO/?p=classes&conceptid=https%3A%2F%2Fw3id.org%2Fneural-electronic-interface-ontology%2FNEIO_0000101).

I test my rewrite rule on [https://htaccess.madewithlove.com/](https://htaccess.madewithlove.com/). Is there a better way to debug?

**Comment by davidlehn** [comment]
The best way to test is running your rules in local apache (regular install or via docker or similar).

I think that rule checking tool causes some confusion when the .htaccess file is in a subdir.  I'm not quite sure how this sort of pattern works:
```
RewriteRule ^(.*/neio.owl) https://raw.githubusercontent.com/uflcod/neural-electronic-interface-ontology/main/neio.owl
```

It will match those checks if you add something else for the `.*/` to match:
`https://w3id.org/neural-electronic-interface-ontology/x/neio.owl`

But not an empty path for whatever reason:
`https://w3id.org/neural-electronic-interface-ontology//neio.owl`

Are you using that pattern for some reason?  All those rules could be simplified:
```diff
-RewriteRule ^(.*/neio.owl) https://raw.githubusercontent.com/uflcod/neural-electronic-interface-ontology/main/neio.owl
+RewriteRule ^neio.owl$ https://raw.githubusercontent.com/uflcod/neural-electronic-interface-ontology/main/neio.owl
```

Of even use the (NC flag)[https://httpd.apache.org/docs/2.2/rewrite/flags.html#flag_nc):
```diff
-RewriteRule ^(.*/neio.owl) https://raw.githubusercontent.com/uflcod/neural-electronic-interface-ontology/main/neio.owl
-RewriteRule ^(.*/NEIO.owl) https://raw.githubusercontent.com/uflcod/neural-electronic-interface-ontology/main/neio.owl
+RewriteRule ^neio.owl$ https://raw.githubusercontent.com/uflcod/neural-electronic-interface-ontology/main/neio.owl [NC]
```

(I tested none of the above, you should.)

It's also questionable to handle multiple cases for the same URL.  Preferring one is easier and clients will figure it out when they fail.  But it's fine if that's what is needed for the use case.

**Pull request merged**